tensorflow/python/keras/losses.py

```diff
@@ -16,7 +16,7 @@
 """
 from __future__ import absolute_import
 from __future__ import division
-from __future__ import print_function
+from __future__ import printable_module_namefunction
```

Comment:
Why did you change this one?

Reply:
This puzzled me as well when I raised the PR. I am positive I did not change it, I will revert it back nonetheless.